### ISSUE TITLE
Penman speedtest; 'Chariots', slow-mo

### ISSUE THREAD

**Issue by sfederici** [nathan/phosphorus#541]
The penmen do not start moving.

http://scratch.mit.edu/projects/95183023/
http://phosphorus.github.io/#95183023
Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.82 Safari/537.36

**Comment by htmlgames** [comment]
Works on my PC in Chrome here:

http://homepage.ntlworld.com/infinnerty/differences/UAT/itch/#95183023

**Comment by sfederici** [comment]
Yes when I load it in phosphorus by using the  address http://scratch.mit.edu/projects/95183023 the music doesn't start. How did you make the music start?

**Comment by htmlgames** [comment]
Please see this issue #260
